tf-modules/report-granules/README.md

```diff
@@ -0,0 +1,26 @@
+# Report Executions
+
+This package includes a Lambda function to process granule ingest notifications received via SNS and store it to a database.
+
+## Deployment
+
+1. Copy the .tfvars sample file: `cp terraform.tfvars.sample terraform.tfvars`
+2. Populate the sample file with values that apply to your AWS environment (see configuration variables section below).
+3. Deploy this module: `terraform apply`
+
+NOTE: Terraform will ignore the `aws_profile` config variable if you have static credentials or environment variables set, see the [AWS Provider page](https://www.terraform.io/docs/providers/aws/index.html#authentication).
+
+## Configuration
+
+Configuration variables are shown in `terraform.tfvars.sample`, and are explained below:
+
+```text
```

Comment:
In my current PR, I opted to create a table for documenting the variables instead of a code block. I'm actually not sure if it's better though. I like that it feels less like repeating the contents of `terraform.tfvars.sample`, but it's a bit more onerous to manage for developers

Reply:
I think I'd prefer the consistency. Yes it kind of repeats `terraform.tfvars.sample`, but it also gives it context, which the raw file doesn't necessarily do.